lib/movie.dart

```diff
@@ -0,0 +1,37 @@
+class Movie {
```

Comment:
format the code with the shortcut and add the missing extra line at the end on file